e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -178,12 +178,48 @@ describe("scenarios > collection defaults", () => {
     });
   });
 
-  describe("render last edited by when names are null", () => {
-    beforeEach(() => {
-      restore();
-      cy.signInAsAdmin();
-    });
+  describe("description", () => {
+    it('should display collection description', () => {
+      visitRootCollection();
+    
+      cy.get('table').within(() => {
+        cy.findByText("First collection").parent().parent().within(() => {
+          cy.icon("info").trigger("mouseenter"); 
+        })
+      })
+    
+      popover().within(() => {
+        cy.findByText(/Collection First/);
+      })
+    })
 
+    it('should support markdown in collection description', () => {
+      getCollectionIdFromSlug("first_collection", id => {
+        visitCollection(id);
+      });
+
+      cy
+        .findByPlaceholderText("Add description")
+        .clear()
+        .type("**important text** and [link](https://metabase.com)")
+        .blur();
```

Comment:
How important is it that we're adding a description manually? We could cut some CI time if we create an item we need via API request

Reply:
There is, we try to keep an eye on that and use the API whenever it makes sense. It's pretty fast locally, but when we're running the full test suite in the CI environment, optimizations like that can make a difference when stacked together

Reply:
replaced DOM interactions with `cy.request`